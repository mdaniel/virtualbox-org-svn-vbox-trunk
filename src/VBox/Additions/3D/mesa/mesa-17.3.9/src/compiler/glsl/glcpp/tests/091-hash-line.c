#line 0
#error line 0 error
#line 25
#error line 25 error
#line 0 1
#error source 1, line 0 error
#line 30 2
#error source 2, line 30 error
#line 45 2 /* A line with a comment */
#define NINETY 90
#define TWO 2
#line NINETY TWO /* A #line line with macro expansion */
#define FUNCTION_LIKE_MACRO(source, line) source line
#line FUNCTION_LIKE_MACRO(180,2)
