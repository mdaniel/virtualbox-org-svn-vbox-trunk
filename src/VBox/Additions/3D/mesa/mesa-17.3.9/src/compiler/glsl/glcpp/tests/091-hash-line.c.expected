0:0(1): preprocessor error: #error line 0 error
0:25(1): preprocessor error: #error line 25 error
1:0(1): preprocessor error: #error source 1, line 0 error
2:30(1): preprocessor error: #error source 2, line 30 error
#line 0
#line 25
#line 0 1
#line 30 2
#line 45 2


#line 90 2

#line 180 2
